src/boot/efi/boot.c

```diff
@@ -2142,7 +2142,7 @@ EFI_STATUS efi_main(EFI_HANDLE image, EFI_SYSTEM_TABLE *sys_table) {
         config_load_defaults(&config, root_dir);
 
         /* scan /EFI/Linux/ directory */
-        config_entry_add_linux(&config, loaded_image, root_dir);
+        config_entry_add_linux(&config, loaded_image->DeviceHandle, root_dir);
 
         /* scan /loader/entries/\*.conf files */
```

Comment:
Stray backslash?

Reply:
i guess, it has been in the sources for about forever. will sneak a fix in

Reply:
hmm, nope, it's on purpose, otherwise gcc complains that /* is contained within a comment...